unifi: 8.5.6 -> 8.6.9

djds opened the pull request:
* https://community.ui.com/releases/UniFi-Network-Application-8-6-9/e4bd3f71-a2c4-4c98-b12a-a8b0b1c2178e
* Move package to pkgs/by-name/un/unifi
* Update aliases

jopejoe1 reviewed (commented):
Could you please split moving to pkgs/by-name into a separate commit?

djds commented:
> Could you please split moving to pkgs/by-name into a separate commit?

I can but why?

jopejoe1 commented:
So that git blame and others recognize that it was the same file

djds committed:
unifi: 8.5.6 -> 8.6.9

https://community.ui.com/releases/UniFi-Network-Application-8-6-9/e4bd3f71-a2c4-4c98-b12a-a8b0b1c2178e

djds committed:
unifi: Update aliases

djds committed:
unifi: Move package to pkgs/by-name/un/unifi

djds commented:
> So that git blame and others recognize that it was the same file

This is done.

Patryk27 commented on pkgs/by-name/un/unifi/package.nix:
```diff
@@ -0,0 +1,53 @@
+    unifi = nixosTests.unifi;
+  };
+
+  meta = with lib; {
+    homepage = "http://www.ubnt.com/";
+    description = "Controller for Ubiquiti UniFi access points";
+    sourceProvenance = with sourceTypes; [ binaryBytecode ];
+    license = licenses.unfree;
+    platforms = platforms.unix;
+    maintainers = with maintainers; [
+      globin
+      patryk27
+    ];
```
`with lib;` is the more common pattern and since both snippets do the same thing, I don't think it's worth changing it here - you'll say `don't use with lib;`, in two weeks someone will say `use with lib;` etc.; unless there's a specific style recommendation, I'd keep whatever's there already.

djds commented:
I can certainly make these changes, but I pretty much straight ported over the existing package with minimal changes. If updating should include a bunch of essentially cosmetic changes, that's fine, but it seems like a bit of a nit to hold up merging this.

jopejoe1 reviewed (approved):
Most of those changes would indeed be nice, but not really required, to change, as they were already like that previously to moving the file.
